woocommerce/wc-admin
# Add manage orders on the go admin note

Fixes #4226

Adds an admin note prompting the user to "manage orders on the go" with an action to download the mobile app.

Note that this also adds the new admin note as a condition to a couple of other admin notes which should only be added if the previous mobile-related admin notes have not been previously actioned.

### Screenshots
![image](https://user-images.githubusercontent.com/224531/93547110-a4ebe880-f9a7-11ea-9499-c8dfc9912ce7.png)

### Detailed test instructions:

1. Use a store that is less than 6 months old
1. Run the `wc_admin_daily` cron task
1. Note that the new note should not appear
1. Make the store more than 6 months old
1. Run the `wc_admin_daily` cron task
1. Note that the new note should have now appeared
1. Delete the note from the database
1. Action one of the earlier notes ("Install Woo mobile app" or "Get real-time order alerts anywhere")
1. Run the `wc_admin_daily` cron task
1. Note that the new note should not appear

### Changelog Note:

Dev: Add manage orders on the go admin note
